### PR TITLE
Added Managing security policies topic

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Based on the following content:
- https://help.stackrox.com/docs/manage-security-policies/use-out-of-box-policies/
- https://help.stackrox.com/docs/manage-security-policies/create-custom-policies/
- https://help.stackrox.com/docs/manage-security-policies/create-custom-policies/
- https://help.stackrox.com/docs/manage-security-policies/create-custom-policies/
- https://help.stackrox.com/docs/manage-security-policies/create-custom-policies/

Preview at https://openshift-docs-git-manage-security-policies-gnelson.vercel.app/openshift-acs/master/operating/manage-security-policies.html

### CODE REVIEW

File: modules/policy-criteria.adoc [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+| ✓
+| ✕
+| ✓
+| Deploy
+
+| Disallowed Image Label
+| Check for the presence of a Docker image label that should not be in use.
+The policy triggers if any image in the deployment has the specified label.
+You can use regular expressions (for both `key` and `value` fields) to match labels.
+The `Disallowed Image Label` policy criteria only works when you are using {product-title} version 3.0.40 or newer and link:/docs/integrate-with-other-tools/integrate-with-image-registries/[integrate with a Docker registry].
```
Comment (bergerhoffer) [inline modules/policy-criteria.adoc:151]: Remove xref

File: modules/policy-criteria.adoc [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+| ✓
+| ✕
+| ✓
+| Deploy
+
+| Required Image Label
+| Check for the presence of a required Docker image label.
+The policy triggers if any image in the deployment does not have the specified label.
+You can use regular expressions (for both `key` and `value` fields) to match labels.
+The `Required Image Label` policy criteria only works when you are using {product-title} version 3.0.40 or newer and link:/docs/integrate-with-other-tools/integrate-with-image-registries/[integrate with a Docker registry].
```
Comment (bergerhoffer) [inline modules/policy-criteria.adoc:161]: Remove xref

File: modules/policy-criteria.adoc [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+
+| Writable Host Mount
+| Resource has mounted a path on the host with write permissions.
+| ✕
+| ✕
+| ✕
+| Deploy
+
+| Unexpected Process Executed
+| Check deployments for which process executions are not listed in the deployment's link:/docs/evaluate-security-risks/use-process-baselining/[locked process baseline].
```
Comment (bergerhoffer) [inline modules/policy-criteria.adoc:309]: Remove xref

File: modules/use-out-of-box-security-policies.adoc [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+* On the RHACS portal, navigate to *Platform Configuration* -> *System policies*.
+
+The Policies view lists the default policies and includes the following parameters for each policy:
+
+* *Name*: A short name for the policy.
+* *Description*: A longer, more detailed description of the alert for the policy.
+* *Lifecycle*: Corresponds to which phase of the container lifecycle, build, deploy, or runtime, does this policy applies, and the phase at which enforcement will be applied, when the policy is enabled.
+* *Severity*: A ranking of the policy, either Critical, High, Medium, or Low, for the amount of attention required.
+
+The default policies have pre-configured parameters and belong to categories such as:
```
Comment (bergerhoffer) [inline modules/use-out-of-box-security-policies.adoc:22]: I don't think a hyphen is needed in preconfigured

File: modules/use-out-of-box-security-policies.adoc [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+* Network Tools
+* Package Management
+* Privileges
+* Security Best Practices
+* System Modification
+* Vulnerability Management
+
+You can edit these categories create your own categories.
+//Add link create your own categories
+When you create your own category, a new widget displays information about that category on the Dashboard.
```
Comment (bergerhoffer) [inline modules/use-out-of-box-security-policies.adoc:37]: Dashboard -> dashboard?

File: operating/manage-security-policies.adoc [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+include::modules/use-out-of-box-security-policies.adoc[leveloffset=+1]
+
+include::modules/modify-existing-security-policies.adoc[leveloffset=+1]
+
+include::modules/create-policy-categories.adoc[leveloffset=+1]
+
+[id="create-custom-policies"]
+== Creating custom policies
+
+In addition to using the default out-of-box policies, you can also create custom policies in {product-title}.
```
Comment (bergerhoffer) [inline operating/manage-security-policies.adoc:21]: Same about "out-of-box"

File: operating/manage-security-policies.adoc [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+To build a new policy, you can clone an existing policy, or create a new one from scratch.
```
Comment (bergerhoffer) [inline operating/manage-security-policies.adoc:23]: Comma isn't necessary

File: operating/manage-security-policies.adoc [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+Beginning from {product-title} version 3.0.44,  you can share your security policies between different Central instances, by exporting and importing policies.
+It helps you enforce the same standards for all your clusters.
+To share policies, you export them as JSON files, and then import them back into another Central instance.
+
+[NOTE]
+====
+Currently, you cannot export multiple security policies at once by using the RHACS portal.
+However, you can use the API for exporting multiple security policies.
+On the RHACS portal, navigate to *Help* > *API reference* to see the API reference.
```
Comment (bergerhoffer) [inline operating/manage-security-policies.adoc:47]: `>` -> `->`

File: operating/manage-security-policies.adoc [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+In addition to using the default out-of-box policies, you can also create custom policies in {product-title}.
+
+To build a new policy, you can clone an existing policy, or create a new one from scratch.
+
+* You can also create policies based on the filter criteria in the *Risk* view in the RHACS portal.
+* You can also use `AND`, `OR`, and `NOT` logical operators for policy criteria to create advanced policies.
+
+include::modules/create-policy-from-system-policies-view.adoc[leveloffset=+2]
+
+include::modules/create-policy-from-risk-view.adoc[leveloffset=+2]
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply operating/manage-security-policies.adoc:30]: part of https://github.com/openshift/openshift-docs/pull/33962

**Comment by gaurav-nelson** [comment]
@bergerhoffer 
The included module `create-policy-from-risk-view.adoc` is part of https://github.com/openshift/openshift-docs/pull/33962 so please merge this after merging 33962

**Comment by gaurav-nelson** [comment]
@bergerhoffer #33962 is merged, this is ready to go.

File: modules/create-policy-from-system-policies-view.adoc [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+You can add multiple scopes and also use regular expressions in link:https://github.com/google/re2/wiki/Syntax[RE2 Syntax] for namespaces and labels.
+However, you cannot use regular expressions for selecting deployments.
+** For *Excluded Images (Build Lifecycle only)*, select all images that you do not want to trigger a violation for.
++
+[NOTE]
+====
+The *Excluded Images* setting only applies when you check images in a continuous integration system with the *Build* lifecycle stage.
+It will not have any effect if you use this policy to check running deployments in the *Deploy* lifecycle stage or runtime activities in the *Runtime* lifecycle stage.
+====
+. In the *Policy Criteria* section, that you you want to trigger the policy for.
```
Comment (bergerhoffer) [inline modules/create-policy-from-system-policies-view.adoc:46]: I think you might have removed some words here. "configure the attributes that you you want to trigger the policy for" ?

File: modules/use-default-security-policies.adoc [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+* On the RHACS portal, navigate to *Platform Configuration* -> *System policies*.
+
+The *Policies* view lists the default policies and includes the following parameters for each policy:
+
+* *Name*: A short name for the policy.
+* *Description*: A longer, more detailed description of the alert for the policy.
+* *Lifecycle*: Corresponds to the phase of the container lifecycle (build, deploy, or runtime) that this policy applies to, and the phase at which enforcement will be applied, when the policy is enabled.
+* *Severity*: A ranking of the policy, either critical, high, medium, or low, for the amount of attention required.
+
+The default policies have pre configured parameters and belong to categories such as:
```
Comment (bergerhoffer) [inline modules/use-default-security-policies.adoc:22]: "pre configured" -> "preconfigured"

File: operating/manage-security-policies.adoc [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+include::modules/modify-existing-security-policies.adoc[leveloffset=+1]
+
+include::modules/create-policy-categories.adoc[leveloffset=+1]
+
+[id="create-custom-policies"]
+== Creating custom policies
+
+In addition to using the default policies, you can also create custom policies in {product-title}.
+
+To build a new policy you can clone an existing policy, or create a new one from scratch.
```
Comment (bergerhoffer) [inline operating/manage-security-policies.adoc:23]: Sorry, should have been more specific. The second comma wasn't necessary. There should have still been one after "To build a new policy":

```
To build a new policy, you can clone an existing policy or create a new one from scratch.
```

**Review by bergerhoffer — COMMENTED** [review]
Just a couple last things, and a conflict to fix, then ready to merge

**Comment by bergerhoffer** [comment]
LGTM

**Pull request merged**